packages/azure-functions/README.md

```
@@ -2,11 +2,56 @@
 
 [![Apache-2.0 licensed][badge-license]][url-license]
 
+Serverless Azure Functions backend for Esposter. Handles asynchronous workloads triggered by Azure EventGrid events — push notifications, webhook delivery, and friend request notifications.
+
 ### Table of Contents
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)